### PR TITLE
Docs: adds v22 Core upgrade guide

### PR DESCRIPTION
**Pull request by ZPain8464**

### CODE REVIEW

File: content/docs/releases/upgrading.mdx [hunk 1]
```diff
@@ -13,6 +13,18 @@ sidebar_position: 10
 
 # Upgrade guide
 
+## Since 0.22.0
+
+### New
+
+#### Hosted Authenticate Service
+
+The [Hosted Authenticate Service](/docs/capabilities/hosted-authenticate-service.md) provides a hosted authenticate service URL and hosted identity provider. The [Self-Hosted Authenticate Service](/docs/capabilities/self-hosted-authenticate-service), which requires a self-hosted authenticate service URL and identity provider, is still supported for users that prefer to host these services themselves.
```
Comment (desimone) [inline content/docs/releases/upgrading.mdx:22]:
```suggestion
The [Hosted Authenticate Service](/docs/capabilities/hosted-authenticate-service.md) replaces the need for an explicitly authenticate service URL / hosted identity provider. Users can now get started without needing their own identity provider, or authenticate service. 

The [Self-Hosted Authenticate Service](/docs/capabilities/self-hosted-authenticate-service), which requires a self-hosted authenticate service URL and identity provider, is still supported for users that prefer to host these services themselves.
```

Clear-er-er?

**Review by desimone — CHANGES_REQUESTED** [review]
Missing the new wildcard functionality and a callout that it is `beta` support

**Comment by ZPain8464** [comment]
@desimone sorry for some reason I thought it wildcard support was only for Enterprise. I'll add.

**Review by calebdoxsey — APPROVED** [review]

File: content/docs/releases/upgrading.mdx [hunk 2]
```diff
@@ -13,6 +13,24 @@ sidebar_position: 10
 
 # Upgrade guide
 
+## Since 0.22.0
+
+### New
+
+#### Hosted Authenticate Service
+
+The [Hosted Authenticate Service](/docs/capabilities/hosted-authenticate-service.md) replaces the need for an explicit authenticate service URL and hosted identity provider. Users can now get started without including their own identity provider, or authenticate service.
```
Comment (desimone) [inline content/docs/releases/upgrading.mdx:22]:
```suggestion
The [Hosted Authenticate Service](/docs/capabilities/hosted-authenticate-service.md) replaces the need for a self-hosted authenticate service and explicit third-party identity provider configuration. Users can now get started without including their own identity provider, or authenticate service.
```

File: content/docs/releases/upgrading.mdx [hunk 3]
```diff
@@ -13,6 +13,24 @@ sidebar_position: 10
...
+
+The [Self-Hosted Authenticate Service](/docs/capabilities/self-hosted-authenticate-service), which requires a self-hosted authenticate service URL and identity provider, is still supported for users that prefer to host these services themselves.
```
Comment (desimone) [inline content/docs/releases/upgrading.mdx:24]: The sentence is kinda awkward.

File: content/docs/releases/upgrading.mdx [hunk 4]
```diff
@@ -13,6 +13,24 @@ sidebar_position: 10
...
+
+#### Wildcard From Routes
+
+[Wildcard From Routes](/docs/reference/routes/from#wildcard-from-routes) support allows you to define multiple near-identical external `from` routes that point to the same internal `to` route. Wildcard From Routes is a Beta feature.
```
Comment (desimone) [inline content/docs/releases/upgrading.mdx:28]: > Wildcard From Routes

Is awkwardly written. The feature allows wildcard syntax (e.g. `*`) in the `From` route configuration option.

**Review by desimone — APPROVED** [review]

**Pull request merged**